<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>465</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="principal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbRoom">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>1001</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>QPushButton {
  align-items: center;
  background-color: rgb(238, 139, 0);
  border-radius: 6px;
  box-shadow: transparent 0 0 0 3px,rgba(18, 18, 18, .1) 0 6px 20px;
  box-sizing: border-box;
  color: rgb(255, 255, 255);
  cursor: pointer;
  display: inline-flex;
  flex: 1 1 auto;
  font-family: Inter,sans-serif;
  font-size: 1.2rem;
  font-weight: 700;
  justify-content: center;
  line-height: 1;
  margin: 0;
  outline: none;
  padding: 1rem 1.2rem;
  text-align: center;
  text-decoration: none;
  transition: box-shadow .2s,-webkit-box-shadow .2s;
  white-space: nowrap;
  border: 0;
  user-select: none;
  touch-action: manipulation;
}

QPushButton:hover {
	box-shadow: #121212 0 0 0 3px, transparent 0 0 0 0;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>room_akinator.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbAki">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>110</y>
       <width>241</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="akinatorimg/imgs.qrc">:/aki_expressions/aki01.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 40px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbQuestion">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 12pt &quot;Segoe UI&quot;;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>321</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="btns_anws">
      <item>
       <widget class="QPushButton" name="btnYes">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
        </property>
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNo">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
        </property>
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnIdk">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
        </property>
        <property name="text">
         <string>I Don't Know</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPs">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
        </property>
        <property name="text">
         <string>Probably</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
        </property>
        <property name="text">
         <string>Probably Not</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>430</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>150</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>170</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="akinatorimg/imgs.qrc"/>
 </resources>
 <connections/>
</ui>
